### DISCUSSION TITLE
Monitoring staking programmatically

### DISCUSSION THREAD

**Discussion by wojciechowskip** [cosmos/community#26] [Chain Builders (Cosmos SDK)]
Guys can I use cosmos SDK to monitor my rewards and staking related information like reward per block etc?
And maybe if it's not possible - is there a public API for retrieving things like this? Like you request staking information based on address passed?

I am new to cosmos so sorry for stupid questions :)

Thanks in advance!

**Comment by michaelbarbas** [comment]
Have you found any more info on this?
